extensions/sovity-messenger/src/main/java/de/sovity/edc/extension/messenger/controller/SovityMessageController.java

```diff
@@ -165,7 +170,7 @@ private SovityMessageResponse buildErrorHandlerExceptionHeader(SovityMessageRequ
         return new SovityMessageResponse(headerStr, "");
     }
 
-    private Handler<Object, Object> getHandler(SovityMessageRequest request) {
+    private HandlerWithClaims<Object, Object> getHandler(SovityMessageRequest request) {
```

Comment:
I don't you need to rename this class. It's still the only handler we have. It just has a slighly different structure. I believe it does not even contain the claims, right? So the name is actually misleading